data/mods/ssb/abilities.ts

```diff
@@ -727,18 +729,51 @@ export const BattleAbilities: {[k: string]: ModdedAbilityData} = {
 		shortDesc: "Changes the pokemon's form upon switch-in depending on the amount of pokemon still alive on the user's team.",
 		name: "The Numbers Game",
 		onStart(pokemon) {
-			if (pokemon.side.pokemonLeft > 3) return;
-			const assignNewMoves = (poke: Pokemon, moves: string[]) => {
-				const carryOver = poke.moveSlots.slice().map(m => {
-					return m.pp / m.maxpp;
-				});
-				// Incase theres ever less than 4 moves
+			/**
+			 * Assigns a new set to a Pokémon
+			 * @param poke the Pokemon to assign the set to
+			 * @param newSet the SSBSet to assign
+			 */
+			const changeSet = (poke: Pokemon, newSet: SSBSet) => {
+				// For some reason EVs and IVs in an SSBSet can be undefined...
+				const evs: StatsTable = {
+					hp: newSet.evs?.hp || poke.set.evs.hp,
+					atk: newSet.evs?.atk || poke.set.evs.atk,
+					def: newSet.evs?.def || poke.set.evs.def,
+					spa: newSet.evs?.spa || poke.set.evs.spa,
+					spd: newSet.evs?.spd || poke.set.evs.spd,
+					spe: newSet.evs?.spe || poke.set.evs.spe,
+				};
```

Comment:
im gonna be honest, this looks like you're going to overflow in legal EVs

Reply:
Yeah, this would overflow if I do the undefined defaulting thing you suggest below. I'll change it.